File: 프로그래머스/1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        String Week[] = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        int Day[] = {31,29,31,30,31,30,31,31,30,31,30,31};
        int sum =0;
        
        for(int i=0; i<a-1; i++){
            sum += Day[i];
        }
        return Week[(((b-1 + sum) % 7))];
    }
}

File: 프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    const Week = ["FRI","SAT","SUN","MON","TUE","WED","THU"];
    const Day = [31,29,31,30,31,30,31,31,30,31,30,31];
    let sum =0;
    
    for(let i=1; i<a; i++){
        sum += Day[i-1];
    }
    
    return Week[(b - 1 + sum) % 7];
}